src/hdusd/mx_nodes/nodes/base_node.py

```diff
@@ -392,17 +435,21 @@ def create_property(mx_param):
         prop_attrs['description'] = mx_param.getAttribute('doc')
 
         if mx_param.hasAttribute('uimin'):
-            prop_attrs['min'] = parse_val(prop_type, mx_param.getAttribute('uimin'), True)
+            prop_attrs['min'] = mx_utils.parse_value_str(
```

Comment:
What do you think on something like that, would it be better or not?

        VALUE_LIMITS_MAPPING = {
            'uimin': 'min',
            'uimax': 'max',
            'uisoftmin': 'soft_min',
            'uisoftmax': 'soft_max',
        }
        
        for mx_attr_name, prop_attr_name in VALUE_LIMITS_MAPPING:
            if mx_param.hasAttribute(mx_attr_name):
                prop_attrs[prop_attr_name] = mx_utils.parse_value_str(
                    mx_param.getAttribute(mx_attr_name), mx_type, first_only=True)

Reply:
Yes, this would be better, I've made changes.